src/utils.ts

```diff
@@ -28,3 +28,9 @@ export const getWebchatButtonLabel: getWebchatButtonLabel = button => {
 	}
 	return title;
 };
+
+export const getRandomId = (prefix = "") => {
+	const id = self?.crypto?.randomUUID() || Date.now();
+
+	return `${prefix}-${id}`;
```

Comment:
empty prefix will result in a id like `-someID` is that expected?

Reply:
actually we use that function always with prefix, but since the default is empty string would be coherent to handle also the empty case. I will fix this on the next commit.